airbyte-integrations/connectors/source-pokeapi/source_pokeapi/schemas/pokemon.json

```diff
@@ -3,45 +3,51 @@
   "type": "object",
   "properties": {
     "id": {
-      "type": ["null", "integer"]
```

Comment:
`null` is redundant: these properties aren't `required`, so the source can just not set them at all.

If we find that sources actually want to output `{"id": null}`, then we can always update the well-known types to use `type: [null, integer]`.